tox.ini

```diff
@@ -38,8 +41,10 @@ setenv =
 sitepackages=true
 commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git@{posargs:main}
-    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/dataloader.git@{posargs:main}
+    #python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/dataloader.git@{posargs:main}
+    python -m pip install --upgrade git+https://github.com/bschifferer/dataloader.git@change_output
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/nvtabular.git@{posargs:main}
+    python -m pip install --upgrade git+https://github.com/bschifferer/dataloader.git@change_output
     horovodrun -np 2 sh examples/usecases/multi-gpu/hvd_wrapper.sh pytest -m horovod -rxs tests/unit
```

Comment:
I noticed that the multi-gpu test is failing. Can we try `python -m pytest` here? i.e., `horovodrun -np 2 sh examples/usecases/multi-gpu/hvd_wrapper.sh python -m pytest -m horovod -rxs tests/unit`

Reply:
Thanks @edknv . Let me try that

Reply:
Looks like this hits version conflicts on Tensorflow. Maybe we have different versions installed on the CI container vs in the Tox environment?

Reply:
PR for bumping the TF version: https://github.com/NVIDIA-Merlin/Merlin/pull/849.

@gabrielspmoreira In the meantime, we could also try removing:
```
deps =
    -rrequirements/test.txt
```
on lines 32-33 to unblock this. Those line will install TF 2.10 in the tox environment, while the ci-runner still has TF 2.9, which seems to be leading to conflict. Maybe worth a try?

Reply:
Just pushed that change to try

Reply:
Yeah @karlhigley . We still have the horovod multi-GPUs tests failing for not using the modified dataloader

Reply:
Sorry I think we should still have `python -m pytest` in the GPU test (AND remove `-rrequirements/test.txt` at the same time). :crossed_fingers: